### PR TITLE
Add help text to name fields

### PR DESCRIPTION
**Pull request by thisisgoldman**

### CODE REVIEW

**Comment by thisisgoldman** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/311?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#311** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/311?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#310** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/310?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#308** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/308?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#307** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/307?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @thisisgoldman and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: assets/svelte/consumers/HttpPullForm.svelte [hunk 1]
```diff
@@ -337,15 +354,20 @@
...
+              id="consumer-name"
+              bind:value={form.name}
+              placeholder="Enter a unique name for your consumer"
+              data-1p-ignore
+              data-lpignore="true"
+              data-form-type="other"
+              disabled={isEditMode}
+            />
+            <p class="text-xs font-light">
+              Alphanumeric separated by underscores. No special characters.
```
Comment (RTLS) [inline assets/svelte/consumers/HttpPullForm.svelte:368]: must contain only alphanumeric characters, underscores, hyphens, or dots

File: assets/svelte/consumers/HttpPushForm.svelte [hunk 2]
```diff
@@ -492,15 +509,20 @@
...
+              id="consumer-name"
+              bind:value={form.name}
+              placeholder="Enter a unique name for your consumer"
+              data-1p-ignore
+              data-lpignore="true"
+              data-form-type="other"
+              disabled={isEditMode}
+            />
+            <p class="text-xs font-light">
+              Alphanumeric separated by underscores. No special characters..
```
Comment (RTLS) [inline assets/svelte/consumers/HttpPushForm.svelte:523]: must contain only alphanumeric characters, underscores, hyphens, or dots

please update changeset.ex as well !

File: assets/svelte/consumers/Wizard.svelte [hunk 3]
```diff
@@ -875,13 +875,19 @@
...
+                  <div class="flex flex-col gap-1">
+                    <Input
+                      id="name"
+                      bind:value={form.name}
+                      placeholder="Enter consumer name"
+                      on:input={() => handleFormUpdate({ name: form.name })}
+                      on:focus={() => (activeInfo = "name")}
+                    />
+                    <p class="text-xs font-light">
+                      Alphanumeric separated by underscores. No special
```
Comment (RTLS) [inline assets/svelte/consumers/Wizard.svelte:887]: ditto

File: assets/svelte/databases/Form.svelte [hunk 4]
```diff
@@ -434,15 +434,20 @@
...
+              type="text"
+              id="name"
+              bind:value={form.name}
+              placeholder="Enter a unique name for your database"
+              data-1p-ignore
+              data-lpignore="true"
+              data-form-type="other"
+            />
+            <p class="text-xs font-light">
+              Alphanumeric separated by underscores. No special characters.
```
Comment (RTLS) [inline assets/svelte/databases/Form.svelte:448]: ditto

Comment (RTLS) [reply assets/svelte/databases/Form.svelte:448]: const nameError in constants.js

**Review by RTLS — APPROVED** [review]

**Comment by thisisgoldman** [comment]
### Merge activity

* **Sep 24, 8:06 PM EDT**: @thisisgoldman started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/sequinstream/sequin/311).

**Pull request merged**